### ISSUE TITLE
Add version switch

### ISSUE THREAD

**Issue by jftuga** [jesusprubio/up#78]
You should consider adding a `-v` switch that will output the program's version and GitHub URL.  Right now, there is no easy way to tell which version of `up` I am running or where I might download a new version.  While `up` is a great program name, it does make it a bit hard to perform a web search for since it is such a generic word.  Sometimes `-v` is used for `verbose`, but since you already have a `-dbg` option, I think it would be OK to use `-v` for version information.

**Example:**

```shell
$ up -v
up v1.2.1
https://github.com/jesusprubio/up
```

**Comment by jesusprubio** [comment]
Yes, we need this. In fact I have changed `-dbg` to `-vv` to keep the `-v` free.
